cookbooks/aws-parallelcluster-common/spec/unit/resources/lustre_setup_spec.rb

```diff
@@ -170,51 +170,35 @@ def self.setup(chef_run)
     end
   end
 
-  context "on redhat with kernel 4.18.0-425.3.1.el8" do
-    cached(:chef_run) do
-      runner = ChefSpec::Runner.new(
-        platform: 'redhat', version: '8',
-        step_into: ['lustre']
-      ) do |node|
-        node.automatic['platform_version'] = "8.2"
-        node.override['cluster']['kernel_release'] = "anything 4.18.0-425.3.1.el8 something"
-      end
-      Lustre.setup(runner)
-    end
-
-    it 'can not install lustre' do
-      is_expected.to write_log("FSx for Lustre is not supported in kernel version 4.18.0-425.3.1.el8 of RHEL, please update the kernel version")
-        .with(level: :warn)
-    end
-  end
-
-  context "on redhat supporting lustre" do
-    cached(:chef_run) do
-      runner = ChefSpec::Runner.new(
-        platform: 'redhat', version: '8',
-        step_into: ['lustre']
-      ) do |node|
-        node.automatic['platform_version'] = "8.2"
-        node.override['cluster']['kernel_release'] = "supported"
+  [%w(193 2), %w(240 3), %w(305 4), %w(348 5), %w(372 6), %w(425 7)].each do |kernel_patch, minor_version|
+    context "on redhat with kernel greater equal than 4.18.0-#{kernel_patch}.3.1.el8 supporting lustre" do
+      cached(:chef_run) do
+        runner = ChefSpec::Runner.new(
+          platform: 'redhat', version: '8',
+          step_into: ['lustre']
+        ) do |node|
+          node.automatic['platform_version'] = "8.2"
```

Comment:
Is there any reason why it is `8.2` instead of `8.7`?

Reply:
No need of the platform_version anymore, we only relay on the kernel version from https://access.redhat.com/articles/3078#RHEL8